### PR TITLE
blender: fix for R1B2 build

### PR DESCRIPTION
**Pull request by kenmays**
Tested on R1B2-hrev54197.

### CODE REVIEW

**Comment by korli** [comment]
What about x86?

File: media-gfx/blender/blender-2.79b.recipe [hunk 1]
```diff
@@ -190,9 +190,9 @@ BUILD()
 		-DZLIB_LIBRARY_RELEASE=`finddir B_SYSTEM_LIB_DIRECTORY`${secondaryArchSubDir}/libz.so.1 \
 		-DWITH_GHOST_SDL=ON \
 		-DWITH_BUILDINFO=OFF \
-		-DWITH_PYTHON_INSTALL=OFF \
-		-DWITH_PYTHON_INSTALL_NUMPY=OFF \
-		-DWITH_PYTHON_INSTALL_REQUESTS=OFF \
+		-DWITH_PYTHON_INSTALL=ON \
+		-DWITH_PYTHON_INSTALL_NUMPY=ON \
+		-DWITH_PYTHON_INSTALL_REQUESTS=ON \
```
Comment (waddlesplash) [inline media-gfx/blender/blender-2.79b.recipe:195]: Why were these off before, but now enabled?

Comment (kenmays) [reply media-gfx/blender/blender-2.79b.recipe:195]: There were some earlier issues with the Python 3.x dependencies which seem resolved now.

**Comment by kenmays** [comment]
@korli - On hrev54185 x86, building OK for me. Need to test on buildbot though.

**Comment by kenmays** [comment]
x86: The -17 recipe seems to break at OpenVDB. Need to review to drop x86 or fix the OpenVDB and other issues for x86. I've tested x86_64 builds (i.e. full-enabled/release) and they work fine.

**Comment by X547** [comment]
It is possible to disable some dependencies only for x86. x86 version should be available.

**Comment by kenmays** [comment]
NOTE: The current recipe works for hrev54194-54202. This fixes Blender 2.79b x86_64 with all main features functional.

**Comment by waddlesplash** [comment]
Supplanted by https://github.com/haikuports/haikuports/commit/88925b78f3c42715e0110018c5e0e17a8e40271e

**Pull request closed without merging**